packages/rpc/test/chainStorageWatcher.test.ts

```diff
@@ -15,12 +15,20 @@ const unmarshal = (val: unknown) => val;
 
 let watcher: ReturnType<typeof makeAgoricChainStorageWatcher>;
 
+vi.mock('../src/marshal', () => ({ makeMarshal: () => {} }));
+
 describe('makeAgoricChainStorageWatcher', () => {
   beforeEach(() => {
     watcher = makeAgoricChainStorageWatcher(
       fakeRpcAddr,
       fakeChainId,
-      unmarshal,
+      undefined,
+      {
+        fromCapData: unmarshal,
+        toCapData: marshal,
+        unserialize: unmarshal,
+        serialize: marshal,
```

Comment:
we could drop these. they're deprecated.

Reply:
Needed to satisfy the type